controllers/update_status.go

```diff
@@ -624,9 +640,77 @@ func validateProcessGroup(ctx context.Context, r *FoundationDBClusterReconciler,
 	processGroupStatus.UpdateCondition(fdbv1beta2.PodFailing, failing, cluster.Status.ProcessGroups, processGroupStatus.ProcessGroupID)
 	processGroupStatus.UpdateCondition(fdbv1beta2.PodPending, false, cluster.Status.ProcessGroups, processGroupStatus.ProcessGroupID)
 
+	disableTaintFeature := false
```

Comment:
Shouldn't this check be moved outside of this method? We are running this method for every process group, so we run this check potentially multiple times to calculate the same value.